_layouts/base.html

```diff
@@ -48,18 +57,18 @@
             <a href="https://devedmonton.com">DES</a> with &hearts;.
           </li>
           <li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
-          <li><a href="./policies/privacy-policy.html">Privacy Policy</a></li>
+          <li><a href="/policies/privacy-policy.html">Privacy Policy</a></li>
         </ul>
       </div>
     </footer>
 
     <!-- Scripts -->
-    <script src="assets/js/jquery.min.js"></script>
-    <script src="assets/js/jquery.scrollex.min.js"></script>
-    <script src="assets/js/jquery.scrolly.min.js"></script>
-    <script src="assets/js/browser.min.js"></script>
-    <script src="assets/js/breakpoints.min.js"></script>
-    <script src="assets/js/util.js"></script>
-    <script src="assets/js/main.js"></script>
+    <script src="/assets/js/jquery.min.js"></script>
+    <script src="/assets/js/jquery.scrollex.min.js"></script>
+    <script src="/assets/js/jquery.scrolly.min.js"></script>
+    <script src="/assets/js/browser.min.js"></script>
+    <script src="/assets/js/breakpoints.min.js"></script>
+    <script src="/assets/js/util.js"></script>
+    <script src="/assets/js/main.js"></script>
```

Comment:
Manually tested conversions across all currently-existing pages and it seems to work.  Cleared my browser cache and cleaned `_site` each time as well just in case.

Reply:
Again, we don't want to hard code these to the root as that assumes we'll always serve from there.

Instead, we can use `relative_url` or avoid putting the documents in a subfolder.

Reply:
Ah sorry... I'm a rebase sort of person so the "View changes" link above won't work anymore.  But I am pretty sure I corrected all these now!

Reply:
(GitLab doesn't seem to lose track of these comments FWIW.  I just assumed GitHub did the same thing)